Objects/unicodeobject.c

```diff
@@ -683,13 +683,38 @@ backslashreplace(_PyBytesWriter *writer, char *str,
     return str;
 }
 
+static Py_ssize_t
+xmlcharrefreplace_get_incr(Py_UCS4 ch)
+{
+    Py_ssize_t incr;
+
+    if (ch < 10)
+        incr = 2+1+1;
+    else if (ch < 100)
+        incr = 2+2+1;
+    else if (ch < 1000)
+        incr = 2+3+1;
+    else if (ch < 10000)
+        incr = 2+4+1;
+    else if (ch < 100000)
+        incr = 2+5+1;
+    else if (ch < 1000000)
+        incr = 2+6+1;
```

Comment:
Please format this according to PEP-7.

Reply:
Sure! I've just copy pasted it from here: https://github.com/python/cpython/pull/101729/files#diff-34c966e7876d6f8bf801dd51896327e4f68bba02cddb95fbf3963f0b2e39c38aL706

Reply:
I know, but _the boys scout rule_ :)